ci/playbooks/architecture_crc/run-architecture-crc.yml

```diff
@@ -0,0 +1,72 @@
+---
+# Usage and expected parameters
+# $ ansible-playbook run-architecture-crc.yml \
+#     -e cifmw_architecture_repo=$HOME/architecture \
+#     -e cifmw_architecture_scenario=hci \
+#     -e cifmw_networking_mapper_networking_env_def_path=$HOME/net-env.yml
+#     [any other parameter/files your VA/DT might need]
+#
+# cifmw_architecture_repo: location of the architecture repository.
+# cifmw_architecture_scenario: the scenario you want to test.
+# cifmw_networking_mapper_networking_env_def_path: path to the
+#        networking-mapper environment definition file.
+#
+# Special parameters set in the playbook (you can override them)
+# cifmw_basedir: defaults to ~/ci-framework-data
+# cifmw_zuul_target_host: target host. Defaults to localhost
+
+- name: Test architecture automations
+  hosts: "{{ cifmw_zuul_target_host | default('controller') }}"
+  gather_facts: true
+  vars:
+    _homedir: "{{ ansible_user_dir | default(lookup('env', 'HOME')) }}"
+    cifmw_basedir: >-
+      {{
+        (_homedir,
+         'ci-framework-data') |
+        path_join
+      }}
+    cifmw_path: >-
+      {{
+        ['~/bin',
+         ansible_env.PATH] | join(':')
+      }}
+    ci_framework_src_dir: "{{ ansible_user_dir }}/src/github.com/openstack-k8s-operators/ci-framework"
+  pre_tasks:
+    - name: Create needed directories
```

Comment:
this could be done via the ci_setup/directories tasks file (you have to provide `directory_state: directory` )

Reply:
it's a good idea, but I'd still need an extra task to create `parameters` dir, since it's not created by the `ci_setup` https://github.com/openstack-k8s-operators/ci-framework/blob/c4ee373c848aa16d1f0767117f61426f17f2a9f4/roles/ci_setup/tasks/directories.yml#L21 and it's needed to store the `interfaces-info.yml` file